lib/popcorn/package.ini

```diff
@@ -0,0 +1,12 @@
+[package]
+name=popcorn
+tags=web,lib
+maintainer=Alexandre Terrasa <alexandre@moz-code.org>
+license=Apache-2.0
+version=1.0
```

Comment:
It says starting with ;)
On May 17, 2016 11:03 AM, "Jean Privat" <notifications@github.com> wrote:

> In lib/popcorn/package.ini
> <https://github.com/nitlang/nit/pull/2094#discussion_r63539134>:
>
> > @@ -0,0 +1,12 @@
> > +[package]
> > +name=popcorn
> > +tags=web,lib
> > +maintainer=Alexandre Terrasa <alexandre@moz-code.org>
> > +license=Apache-2.0
> > +version=1.0
>
>
> https://github.com/nitlang/nit/pull/2094/files#diff-d4e9a818858d1fa3e23075630201d3b2R548
> says version 0.1
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/nitlang/nit/pull/2094/files/a951f2cf22e6f38ae14459caa56b77fd35b75e90#r63539134>
>